eventing/README.md

```diff
@@ -91,7 +91,7 @@ the above sources, and adds GCP PubSub, which requires the listed secret):
 
 ```bash
 kubectl --namespace knative-sources create secret generic gcppubsub-source-key --from-literal=key.json=''
```

Comment:
We don't need this anymore.  This secret is in the release now.

Reply:
The unusable dummy secret? I think I'm actually going to yank this because frankly it causes so many problems because it requires killing the pods, etc.